src/gengowatcher/notifier.py

```
@@ -0,0 +1,92 @@
+    Plays a sound file using system audio players.
+    Tries paplay (PulseAudio/PipeWire), then aplay (ALSA) as fallback.
+    This runs in a separate thread to avoid blocking.
+    """
+    def _play():
+        if not Path(sound_file_path).is_file():
+            logger.warning(f"Sound file not found: {sound_file_path}")
+            return
+
+        # Try paplay first (PulseAudio/PipeWire - common on modern Linux including Arch)
+        try:
+            subprocess.run(
```

Ruff (0.14.10)

`subprocess` call: check for execution of untrusted input

(S603)